src/main/java/Duke.java

```diff
@@ -1,10 +1,234 @@
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.FileWriter;
+
+import Task.Task;
+import todo.todo;
+import Event.Event;
+import Deadline.Deadline;
+import Exception.*;
```

Comment:
Maybe the imported classes can be listed explicitly.
```suggestion
import Exception.noTaskDescriptionException;
import Exception.invalidInputException;
import Exception.dukeException;
```